src/PostgreSQLCopyHelper/PostgreSQLCopyHelper.cs

```diff
@@ -56,12 +56,10 @@ public ValueTask<ulong> SaveAllAsync(NpgsqlConnection connection, IEnumerable<TE
 
         private async ValueTask<ulong> DoSaveAllAsync(NpgsqlConnection connection, IEnumerable<TEntity> entities)
         {
-            using (var binaryCopyWriter = connection.BeginBinaryImport(GetCopyCommand()))
-            {
-                await WriteToStream(binaryCopyWriter, entities);
+            await using var binaryCopyWriter = connection.BeginBinaryImport(GetCopyCommand());
+            await WriteToStream(binaryCopyWriter, entities);
 
-                return await binaryCopyWriter.CompleteAsync();
-            }
+            return await binaryCopyWriter.CompleteAsync();
```

Comment:
We are able to get rid of the using braces with C#8 as well

Reply:
Great!